src/pybind/mgr/dashboard/tox.ini

```diff
@@ -140,3 +142,31 @@ commands =
 [testenv:check]
 commands =
     python ci/check_grafana_uids.py frontend/src/app ../../../../monitoring/grafana/dashboards
+
+[testenv:openapi-{check,fix}]
+basepython = python3
+allowlist_externals = diff
+description =
+    check: Ensure that auto-generated OpenAPI Specification is exactly the same as last version
```

Comment:
How about this?
```suggestion
    check: Ensure that auto-generated OpenAPI Specification matches the current version
```

Reply:
BTW; I think it would be great if all available `tox` commands would have such descriptions.

Reply:
Looks better! Thanks! Yeah, however to display those you need to add `-v` to the `-l` flag : /